### DISCUSSION TITLE
Sticky Views in E4

### DISCUSSION THREAD

**Discussion by laeubi** [eclipse-platform/eclipse.platform.ui#2027] [Q&A]
I just came across that there are "[sticky views](https://help.eclipse.org/latest/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Freference%2Fextension-points%2Forg_eclipse_ui_views.html)" in E3 model but can't find out if there is a counterpart in E4 as well.

**Comment by vogella** [comment]
I'm not sure but usually these flags are handled via persistedState or tags. If you have such a view in your runtime, check via the model spy its additional data:


![image](https://github.com/eclipse-platform/eclipse.platform.ui/assets/139910/5bc3d5dc-bf8a-454f-b32f-6984229c7f60)


![image](https://github.com/eclipse-platform/eclipse.platform.ui/assets/139910/e80a873f-6bb1-4767-b9c6-388457d201fc)
